pkg/apis/v1alpha1/common.go

```diff
@@ -199,12 +199,19 @@ type FieldSelectorRequirement struct {
 	Values []string `json:"values,omitempty"`
 }
 
+type StampedRef struct {
+	*corev1.ObjectReference `json:",inline,omitempty"`
+
+	// Resource refers to the resource name and group [NAME(.GROUP)]
```

Comment:
Perhaps more complete documentation?

```suggestion
	// Resource refers to the resource name and group [NAME(.GROUP)]
	// The NAME segment is the CRD's plural value. You can use this to fully qualify a kubectl reference.
```